flake.nix

```diff
@@ -616,6 +649,14 @@
               chain-spec = "dali-dev";
             }).script;
 
+            # Dali Centauri devnet
+            devnet-dali-centauri = (callPackage mk-devnet {
```

Comment:
can we prefix this like

bridge-devnet-dali

Reply:
fixed